extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -163,7 +156,9 @@ public function testRenderInfoboxDataProvider() {
 						'media-type' => 'image',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "media-type" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)